UI/AutoSuggestSample/AutoSuggestSample/AutoSuggestSample/Platforms/WebAssembly/LinkerConfig.xml

```diff
@@ -0,0 +1,10 @@
+<linker>
+  <assembly fullname="AutoSuggestSample" />
```

Comment:
Original contents should be restored

Reply:
NOTE: Except for `AutoSuggestSample.Wasm` it will be `AutoSuggestSample` as it's already.